### ISSUE TITLE
Add support for the tooltip.trigger = 'selection' option to the GeoChart

### ISSUE THREAD

**Issue by orwant** [google/google-visualization-issues#1534]
```
Currently, the GeoChart supports the values 'focus' and 'none' for this option.
```

Original issue reported on code.google.com by `grabks@google.com` on 2014-03-24 13:35:15

**Comment by orwant** [comment]
```
This would be really useful for me!
```

Original issue reported on code.google.com by `edward.boyle7` on 2014-04-08 09:35:47

**Comment by orwant** [comment]
```
Sounds nice!
```

Original issue reported on code.google.com by `covrig.catalin` on 2014-05-22 09:44:45

**Comment by orwant** [comment]
```
Looks like this is already available in v.1.1 :)
https://groups.google.com/forum/#!topic/google-visualization-api/h1VQKmumvhM
Will be very useful now that the html tooltips are also available. 
```

Original issue reported on code.google.com by `cmoreira.wp` on 2014-05-22 10:15:42

**Comment by orwant** [comment]
```
This is now available in 1.1. There are numerous improvements that we can do, but the
basic behavior is there and will hopefully be useful to everyone. Marking this bug
as fixed. If you find any issues with the behavior, please open a new bug.
```

Original issue reported on code.google.com by `grabks@google.com` on 2014-05-22 20:54:41

**Comment by biswanathsahoo** [comment]
I want to add one specific tooltip to null value area in google geo chart.Is it possible?
